test/models/services.ts

```diff
@@ -1033,4 +1054,54 @@ describe('createWorkflowSteps', function () {
       expect(stagingLocation).to.include('dummy/p1');
     });
   });
+
+  describe('when a collection has matching umm-c conditional native_format', function () {
+    const ummOperation = _.cloneDeep(operation);
+    ummOperation.geojson = 'interesting shape';
+    ummOperation.ummcollections = [{
+      'meta': {
+        'concept-id': 'C1234-TEST',
+      },
+      'umm': {
+        'MetadataSpecification': {
+          'Name': 'UMM-C',
+          'URL': 'https://cdn.earthdata.nasa.gov/umm/collection/v1.17.3',
+          'Version': '1.17.3',
+        },
+        'ArchiveAndDistributionInformation': {
+          'FileArchiveInformation': [ {
+            'Format': 'netCDF-4',
+            'FormatType': 'Native',
+          } ],
+          'FileDistributionInformation': [ {
```

Comment:
Are we making use of this field? If not when you change the type for CmrUmmCollection you may need to drop this field in your test.